contract/__manifest__.py

```diff
@@ -10,6 +10,7 @@
     'license': 'AGPL-3',
     'author': "OpenERP SA,"
```

Comment:
I'm not sure if they have switched already their legal part. They don't put author now in Odoo 10 manifests, so no clue...